bing-dict-cache.el

```diff
@@ -0,0 +1,66 @@
+(defvar bing-dict-cache-auto-save t
+  "Whether to save the bing dict cache when leaving emacs.")
+
+(defvar bing-dict--cache nil
+  "It will be used when offline, its type is a hash table")
+
+(defvar bing-dict-cache-file (expand-file-name "var/bing-dict/bing-dict-save.el" user-emacs-directory)
+  "It is where cache will be stored")
+
+(defvar bing-dict-cache-limit 100
+  "The largest numbers of the items to be cached")
+
+(defun bing-dict--cache-init ()
+  "Initialize bing dict."
+  (setq bing-dict--cache (make-hash-table :test #'equal)))
```

Comment:
As the cache size is only 100, you can actually use the plain old associate list instead of the hash table, and it would be faster (considering the impl in Emacs Lisp) and easy to write code (e.g. no need to do the `hash table` to `associate list` transformation in `bing-dict--update-cache`).

Reply:
Sounds reasonable, and thanks for the useful advice, I will make some improvements.